### PR TITLE
Fix flaky external test

### PR DESCRIPTION
**Pull request by freddyaboulton**
# Description

Fixes flaky test. I think the underlying space must have been restarted?

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have added a short summary of my change to the CHANGELOG.md
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# A note about the CHANGELOG

Hello 👋 and thank you for contributing to Gradio!

All pull requests must update the change log located in CHANGELOG.md, unless the pull request is labeled with the "no-changelog-update" label.

Please add a brief summary of the change to the Upcoming Release > Full Changelog section of the CHANGELOG.md file and include
a link to the PR (formatted in markdown) and a link to your github profile (if you like). For example, "* Added a cool new feature by `[@myusername](link-to-your-github-profile)` in `[PR 11111](https://github.com/gradio-app/gradio/pull/11111)`".

If you would like to elaborate on your change further, feel free to include a longer explanation in the other sections.
If you would like an image/gif/video showcasing your feature, it may be best to edit the CHANGELOG file using the 
GitHub web UI since that lets you upload files directly via drag-and-drop.

### CODE REVIEW

**Review by pngwn — APPROVED** [review]
it passes, so i think that mean it is good!

Thanks @freddyaboulton !

File: CHANGELOG.md [hunk 1]
```diff
@@ -51,8 +51,9 @@ By [@freddyaboulton](https://github.com/freddyaboulton) in [PR 3297](https://git
...
 
 ## Testing and Infrastructure Changes:
-* Adds a script to benchmark the performance of the queue and adds some instructions on how to use it. By [@freddyaboulton](https://github.com/freddyaboulton) and [@abidlabs](https://github.com/abidlabs) in [PR 3272](https://github.com/gradio-app/gradio/pull/3272)
-* Flaky python tests no longer cancel non-flaky tests by [@freddyaboulton](https://github.com/freddyaboulton) in [PR 3344](https://github.com/gradio-app/gradio/pull/3344)  
+- Adds a script to benchmark the performance of the queue and adds some instructions on how to use it. By [@freddyaboulton](https://github.com/freddyaboulton) and [@abidlabs](https://github.com/abidlabs) in [PR 3272](https://github.com/gradio-app/gradio/pull/3272)
+- Flaky python tests no longer cancel non-flaky tests by [@freddyaboulton](https://github.com/freddyaboulton) in [PR 3344](https://github.com/gradio-app/gradio/pull/3344)  
+- Fixed a test failure in `test_external.py` by [@freddyaboulton](https://github.com/freddyaboulton) in [PR 3348](https://github.com/gradio-app/gradio/pull/3348)
```
[the remark this answers is not in the document]
Comment (freddyaboulton) [reply CHANGELOG.md:56]: Makes sense!

**Comment by abidlabs** [comment]
@freddyaboulton instead of querying a temporary file which seemingly changes every time the Space restarts, let's just upload a file to the Space and query that so we don't have this problem in the future

**Comment by abidlabs** [comment]
In fact, such files already exist. What if we just replace the path with:

```py
/proxy=https://gradio-tests-test-loading-examples-private.hf.space/file=Bunny.obj
```

**Comment by freddyaboulton** [comment]
On it

File: gradio/utils.py [hunk 2]
```diff
@@ -886,11 +886,11 @@ def tex2svg(formula, *args):
     svg_start = xml_code.index("<svg ")
     svg_code = xml_code[svg_start:]
     svg_code = re.sub(r"<metadata>.*<\/metadata>", "", svg_code, flags=re.DOTALL)
-    svg_code = re.sub(r' width="[^"]+"', "", svg_code)
+    svg_code = re.sub(r' width="[^"]+"', '', svg_code)
```
Comment (abidlabs) [inline gradio/utils.py:889]: What's going on with `black`??

Comment (freddyaboulton) [reply gradio/utils.py:889]: I don't know 😢

**Comment by abidlabs** [comment]
LGTM thanks for making the update @freddyaboulton!

**Review by abidlabs — APPROVED** [review]

**Comment by freddyaboulton** [comment]
Thank you both for the reviews 🙏

**Pull request merged**